test/dummy/db/migrate/20170205000000_create_users.rb

```diff
@@ -11,5 +11,11 @@ def change
       t.string :name
       t.references :owner, polymorphic: true
     end
+
+    create_table :accounts do |t|
+      t.string :email
+      t.string :merchant_processor
```

Comment:
Hey Chris, 

I had a quick play around with this today.

Can we add this to the billable migration? Users would have to add this column to their billable model but there is no indication about where to do that...

Perhaps adding it to the initial pay:install migration would help.

Reply:
I wasn't sure if we should require a Merchant to also be Billable, that's why it's separate right now. It seems logical that a Merchant would also be Billable, so maybe it's not necessary to separate them out.